docs/GettingStarted.rst

```diff
@@ -1194,6 +1194,25 @@ When you click on the "OK" button a preview of the video is displayed in the con
 
 |image196|
 
+**Insert an attachment**
+
+You can insert attachments in the article's content by clicking in the appopriate icon of the creation form. 
+
+|image199|
+
+In the displayed drawer, you can either upload manually your files or using a drag and drop in the appropriate area. You can add up to 10 files and each file must not exceed 25 Mb.
+
+|image200|
+
+Once you add files, you can see the upload progress in the drawer as well as the number of selected files. You can cancel the upload of a file in progress or delete it if it is already uploaded using the close icon.
+
+|image201|
+
+You can close the drawer using the 'X' icon, the close button or simply by clicking outside the drawer area. If the upload is still in progress, a circular movement will be visible on the icon containing the number of attachments and the post button will be disabled until all files are uploaded.
```

Comment:
```suggestion
You can close the drawer using the 'X' icon, the close button or simply by clicking outside the drawer panel. If the upload is still in progress as you close the panel, it will continue in the background. An animation around the attachments counter badge indicates progress. Note that the post button will be disabled until all files are uploaded.
```